vars/npmExecuteEndToEndTests.groovy

```diff
@@ -95,12 +95,13 @@ void call(Map parameters = [:]) {
                     withCredentials(credentials) {
                         if (appUrl.parameters) {
                             if (appUrl.parameters instanceof List) {
-                                npmExecuteScripts(script: script, parameters: npmParameters, install: false, virtualFrameBuffer: true, runScripts: [config.runScript], scriptOptions: ["--launchUrl=${appUrl.url}"] + appUrl.parameters)
+                                npmExecuteScripts(script: script, parameters: npmParameters, install: false, virtualFrameBuffer: true, runScripts: [config.runScript], scriptOptions: ["--launchUrl=${appUrl.url}"] + appUrl.parameters, buildDescriptorExcludeList: ["**/*/package.json"])
```

Comment:
I would rather propose to make this configurable and provide a default, e.g. later in the stage configuration, maybe also the default from the metadata file is sufficient for now

Reply:
Yes, the default from `npmExecuteScripts` is sufficient, at least for the projects I tested it with. Sure we can also make it configurable.

Reply:
@daniel-kurzynski Is this something which still needs to be addressed?

Reply:
I have addressed it with this commit: https://github.com/SAP/jenkins-library/pull/1800/commits/f93f2573e7671b7a9e9b217efbbae55555004f41